Makefile

```diff
@@ -72,26 +71,19 @@ install: build ## Install the sysl binary into $(GOPATH)/bin
 	cp ./dist/sysl $(GOPATH)/bin
 
 clean: ## Clean temp and build files
-	rm -rf release dist pkg/grammar/temp
+	rm -rf release dist
 
 # Autogen rules
 ANTLR = java -jar pkg/antlr-4.7-complete.jar
-GRAMMARS = pkg/grammar/SyslParser.g4 \
-		   pkg/grammar/SyslLexer.g4
+ANTLR_GO = $(ANTLR) -Dlanguage=Go -lib $(@D) $<
 
-antlr: $(GRAMMARS)
-	$(ANTLR) -Dlanguage=Go -lib pkg/grammar -o pkg/grammar/temp $^
-
-pkg/grammar/sysl_parser.go: antlr
-	cp pkg/grammar/temp/pkg/grammar/sysl*parser*.go pkg/grammar
-	git apply pkg/grammar/antlr4-datarace-fix-parser.go.diff
-
-pkg/grammar/sysl_lexer.go: antlr
-	cp pkg/grammar/temp/pkg/grammar/sysl*lexer*.go pkg/grammar
-	git apply pkg/grammar/antlr4-datarace-fix-lexer.go.diff
+grammar: pkg/grammar/sysl_lexer.go pkg/grammar/sysl_parser.go pkg/parser/grammar.pb.go ## Regenerate the grammars
```

Comment:
pkg/parser/grammar.pb.go can be removed from this line and the file itself removed.

Reply:
Ok done. and force pushed.